src/bot/commands/lastfm/fm.ts

```diff
@@ -164,4 +259,45 @@ export class LastFm extends Command {
 
         await db.save();
     }
+
+    private getAlbumImage(name: string, artist: string, key: string) : Promise<string> {
+        return new Promise(resolve => {
+            let requestURL = request(('http://ws.audioscrobbler.com/2.0/?method=album.search&&album=' + name + '&api_key= ' + key + '&format=json'), (error: any, response: Response, body: any) => {
+                if (error) {
+                    //?
+                }
+
+                let parsed = JSON.parse(body);
+
+                if (parsed.results.albummatches == undefined) {
+                    resolve('https://upload.wikimedia.org/wikipedia/commons/4/48/BLANK_ICON.png');
```

Comment:
If it's just a black image, consider storing it locally so that you don't have to rely on retrieving it from Wikipedia